x/avs/keeper/keeper.go

```diff
@@ -157,3 +171,25 @@ func (k Keeper) DeleteAVSInfo(ctx sdk.Context, addr string) error {
 	store.Delete(avsAddr)
 	return nil
 }
+
+// IteratAVSInfo iterate through avs
+func (k Keeper) IteratAVSInfo(ctx sdk.Context, fn func(index int64, avsInfo types.AVSInfo) (stop bool)) {
```

Comment:
this  **IteratAVSInfo** should be  **IterateAVSInfo**

Reply:
fixed